### PR TITLE
Desktop branch : 

### PR DESCRIPTION
**Pull request by abenezer-addisu**
In this branch 
- i have created the desktop version 
- edit the style with media query

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -101,7 +104,15 @@ form li {
 
 .home p {
   width: 270px;
-  font-size: 16px;
+  font-size: 20px;
+}
+
+/* .connect p {
+  color: #7f8cff;
+} */
```
Comment (Shisui6) [inline CSS/style.css:112]: Please do not leave commented code in your codebase. It makes your code look messy and violates the DRY principle of coding.🙏

File: CSS/style.css
Comment (Shisui6) [inline CSS/style.css:1]: Nice job styling the desktop version✨. However, the styling applied to your mobile version has been affected and the mobile version does not look good anymore. Your webpage is supposed to be responsive on both the mobile and desktop versions. Please take a look at your media queries and make sure that your mobile styling still looks good.

Take a look at the video below for reference.

https://user-images.githubusercontent.com/37116322/230653336-45449a49-ef31-4318-b95a-47d282643cd5.mp4

File: CSS/style.css [hunk 2]
```diff
@@ -830,6 +845,148 @@ hr {
...
+    font-size: 40px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .descr p {
```
Comment (Shisui6) [inline CSS/style.css:913]: Kindly update the styling of each of your cards to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Kindly reduce the font size of the paragraph text
- Kindly increase the spacing between each of the elements in the card details
- Kindly increase the top margin between the title of the card and the top of the card

File: CSS/style.css [hunk 3]
```diff
@@ -270,8 +277,8 @@ h2 {
 .about-myself {
   display: flex;
   flex-direction: column;
-  padding: 114px 24px;
-  gap: 24px;
+  padding: 114px 7px 114px 90px;
+  gap: 0;
   background-color: white;
   border-top-right-radius: 80px;
 }
```
Comment (Shisui6) [inline CSS/style.css:284]: Kindly update the styling of your about myself section to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Kindly make sure the arrow icons on the right are aligned
- Kindly update the images and text in the languages dropdown to be left aligned not center aligned

File: CSS/style.css [hunk 4]
```diff
@@ -830,6 +845,148 @@ hr {
...
+    width: 100%;
+    height: 920px;
+    flex-direction: column;
+    background-image: url("../img/form.png");
+    background-repeat: no-repeat;
+    background-size: inherit;
+    background-position-x: right;
+    background-color: #6070ff;
+    border-top-left-radius: 90px;
+  }
```
Comment (Shisui6) [inline CSS/style.css:890]: Kindly update the styling of your contact section to match the figma design better🙏

Your design:
image

Figma design:
image

A few things to note:
- Please reduce the spacing between the main title, paragraph and form
- Please remove the border radius on the form inputs 
- Please ensure your form inputs have the same font family with the rest of the page

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @abenezer-addisu, 

# Requires Changes 🍀
You have done a great job so far!
Your design is good 🖼️, and your code follows best practices. However, there are a few issues that you still need to work on🪲.

## Highlights
- Good use of media queries ✅
- Good design ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/style.css
Comment (aamirkhan2478) [inline CSS/style.css:1]: [OPTIONAL] Nice job :clap: you implemented every thing the previous reviewer requested well done :+1: but if you check your design there is some space I think on your right side so the vertical scrollbar is shown so kindly fix it :slightly_smiling_face: 

[screen-capture (2).webm](https://user-images.githubusercontent.com/45226848/230716109-6537b102-b5cd-4968-85a9-3a52a184c5dc.webm)

File: CSS/style.css [hunk 5]
```diff
@@ -830,6 +1099,398 @@ hr {
...
+  .languages-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list > ul > li {
+    display: flex;
+
+    /* background: #f7f7f9; */
+    border-radius: 8px;
```
Comment (aamirkhan2478) [inline CSS/style.css:1373]: [OPTIONAL] Please remove all unwanted comments from your codebase it makes your code messy so kindly remove all the unwanted comments like that.

File: index.html [hunk 6]
```diff
@@ -273,35 +273,49 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                      type="text"
+                      name="Message"
+                      maxlength="500"
+                      class="text-area"
+                      placeholder="Write your message here..."
+                      required
+                    />
+                  </li>
+                </ul>
+                <button class="submit">Get in touch</button>
```
Comment (aamirkhan2478) [inline index.html:315]: [OPTIONAL] Please stick to the Figma design your button all effects are not the same as the Figma design kindly fix your all button effects :+1: 

Screenshot from 2023-02-17 17-31-04

**Review by aamirkhan2478 — APPROVED** [review]
Hi @abenezer-addisu , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (abenezer-addisu) [reply index.html:315]: thank you so much i will go through you advice

**Pull request merged**